plugins/TrollManagement/tests/TrollManagementTest.php

```diff
@@ -0,0 +1,188 @@
+<?php
+/**
+ * @author David Barbier<david.barbier@vanillaforums.com>
+ * @copyright 2009-2021 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+namespace VanillaTests\TrollManagement;
+
+use Garden\Container\Container;
+use UserModel;
+use RoleModel;
+use BanModel;
+use VanillaTests\APIv0\TestDispatcher;
+use VanillaTests\SetupTraitsTrait;
+use VanillaTests\SiteTestCase;
+use VanillaTests\TestInstallModel;
+
+/**
+ * Class TrollManagementTest
+ */
+class TrollManagementTest extends SiteTestCase {
+    use SetupTraitsTrait;
+
+    /** @var UserModel */
+    protected $userModel;
+
+    /**
+     * {@inheritDoc}
+     */
+    public function setUp(): void {
+        parent::setUp();
+        $this->createUserFixtures();
+    }
+
+    /**
+     * Setup routine, run before the test class is instantiated.
+     */
+    public static function setupBeforeClass(): void {
+        self::$addons = ['vanilla', 'trollmanagement'];
```

Comment:
Do we need "vanilla" here? If not, removing it and avoiding its setup would be beneficial.

Reply:
Agreed. It is faster.
Done.